idioms-json-2.0/custom_object.json

```diff
@@ -20,7 +20,7 @@
                 "ftp"
             ],
             "modified": "2015-07-31T11:24:39.090Z",
-            "pattern": "[x-elevator-fooz:ftp_command MATCHES 'fp[a-zA-Z0-9]{44}\\\\.bin']",
```

Comment:
The custom observable used to be captured in the past. Because of this changes this is now a `unconvered_term`. Was this intended?

Reply:
same reason as previous comment